benchmarks/ner_embeddings/project.yml

```diff
@@ -0,0 +1,278 @@
+title: "Comparing embedding layers in spaCy"
+description: |
+  This project contains the code to reproduce the results of the
+  [Multi hash embeddings in spaCy](https://arxiv.org/abs/2212.09255) technical report by Explosion.
+  The `project.yml` provides commands to download and preprocess the data sets as well as to
+  run the training and evaluation procedures. Different configuration of `vars` correspond
+  to different experiments in the report.
+
+  There are a few scripts included that were used during the technical report writing process
+  to run experiments in bulk and summarize the results. 
+  The `scripts/run_experiments.py` runs multiple experiments one after the other
+  by constructing and running `spacy project run` commands. The module
+  `scipts/collate_results.py` summarizes the results of the same trials with multiple seeds.
+  Finally, `scripts/plot_results.py` was used to produce the visualizations in the report.
+  These are all small command line apps and you can learn more about the usage as usual with the
+  `--help` flag.
+
+
+vars:
+  ner_config: "multihashembed"
+  dataset: "archaeo"
+  language: "nl"
+  vectors: "nl_core_news_lg"
+  tables_path: "tables"
+  metrics_dir: "metrics"
+  attrs: null
+  rows: "[5000, 2500, 2500, 2500]"
+  include_static_vectors: true
+  min_freq: 10
+  gpu_id: 0
+  seed: 42
+  batch_size: 1000
+  num_hashes: 4
+
+directories:
+  - "assets"
+  - "configs"
+  - "corpus"
+  - "scripts"
+  - "training"
+  - "metrics"
+  - "vectors"
+  - "tables"
+  - "debug"
+
+workflows:
+  setup:
+    - "download-models"
+    - "init-fasttext"
+    - "prepare-datasets"
+    - "make-tables"
+
+  trial:
+    - "init-labels"
+    - "train"
+    - "evaluate"
+    - "evaluate-seen-unseen"
+
+assets:
+  - dest: "assets/fasttext.en.gz"
+    description: "English fastText vectors."
+    url: https://dl.fbaipublicfiles.com/fasttext/vectors-crawl/cc.en.300.vec.gz
+  - dest: "assets/fasttext.es.gz"
+    description: "Spanish fastText vectors."
+    url: https://dl.fbaipublicfiles.com/fasttext/vectors-crawl/cc.es.300.vec.gz
+  - dest: "assets/fasttext.nl.gz"
+    description: "Dutch fastText vectors."
+    url: https://dl.fbaipublicfiles.com/fasttext/vectors-crawl/cc.nl.300.vec.gz
+  - dest: "span-labeling-datasets"
+    git:
+      repo: "https://github.com/explosion/projects"
+      branch: "v3"
+      path: "benchmarks/span-labeling-datasets"
```

Comment:
It's good to have a way that this works directly from the commands in the config, but the dependence on the other project still feels a little awkward. I think for now this is fine to keep as such, as I don't want to introduce more work and delay the blog post further. In the future though, the approach I'd recommend would be 
- pre-calculate the files that the other project would create, and attach them to a release on Github
- fetch those files from within the `assets` command of this project, so that you don't make users re-run this
- having this isolated into the `assets` command also means that users can easily skip this step (and the cloning) if they already had the project/files on disk

Reply:
I agree, it would have been nicer to just attach the files!